coalib/bearlib/aspects/Smell.py

```diff
@@ -3,8 +3,7 @@
 
 @Root.subaspect
 class Smell:
-    """
-    This aspect detects `code smells` or `bad smells` in your code.
+    """Illustrate code smells.
```

Comment:
Again... No `Illustrate`! :) Whole line kind of useless again. Just start with text below